src/lib.rs

```diff
@@ -145,7 +145,8 @@ impl Client {
     /// ```
     pub fn new(options: Options) -> io::Result<Self> {
         UdpSocket::bind(options.from_addr.as_str()).map(move |socket| {
-            let to_addr = options.to_addr.parse::<SocketAddr>().unwrap();
+            let to_addr: Vec<SocketAddr> = options.to_addr.as_str().to_socket_addrs()
```

Comment:
Is the `as_str()` necessary here? I'd think that the `impl Deref<str>` would make it work for `String`. Could be mistaken tho...